recipes/harvesttools/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.2" %}
+{% set base = "https://github.com/marbl/harvest-tools/releases/download" %}
+{% set sha256linux = "772e86fa28a811caed4f07057eba0b3dbb791ba718940c7b2e453d643d78d5d0" %}
+{% set sha256osx = "eff4cdd6971357434a64c05d6d25a1d8932508311f5bd9bbb1f759c6b53a47f2" %}
+
+package:
+  name: harvesttools
+  version: '{{version}}'
+
+source:
+  url: '{{base}}/v{{version}}/harvesttools-Linux64-v{{version}}.tar.gz'   # [linux]
```

Comment:
It would be good if you could compile from source.

Reply:
Issues with gcc versioning with protobuf library dependency when compiling from source.